modules/release-notes/single-source-release-notes.jira2asciidoc.yml

```diff
@@ -13,7 +13,7 @@ sections:
       AND status in (Closed, "Release Pending")
       AND "Release Note Type" in ("Feature", "Enhancement")
       AND fixVersion >= "{version_minor}"
-      AND fixVersion <= "{version_patch}"
+      AND fixVersion < "{version_next}"
```

Comment:
this is better so that we don't have to move things to an arbitrary 1.4.z version for Known Issues